// Copyright 2016 The Chromium OS Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";
option optimize_for = LITE_RUNTIME;

package imageloader;

// Optional fields are considered best practice, but since the client and server
// come from the same code base, this uses required fields to save on
// validation.

message MountImage {
  required string mount_path = 1;
  required string table = 2;
  enum FileSystem {
    SQUASH = 0;
    EXT4 = 1;
  }
  required FileSystem fs_type = 3;
}

message MountResponse {
  required bool success = 1;
}
